src/components/pt-element/PtElementInfo.tsx

```diff
@@ -1,41 +1,36 @@
-import autobind from "autobind-decorator";
-import classNames from "classnames";
 import * as React from "react";
+import classNames from "classnames";
+
 import { IElement } from "../../Element";
 import { getElementLocales } from "../../ElementManager";
 import Button from "../shared/button/Button";
 import "./PtElement.scss";
-
 export interface IPtElementInfoProps {
   element: IElement;
   onClick?: (element: IElement) => void;
 }
 
-@autobind
-class PtElementInfo extends React.Component<IPtElementInfoProps> {
-  public render() {
-    const { element } = this.props;
-    const elementLocales = getElementLocales(element);
+function PtElementInfo({ element, onClick }: IPtElementInfoProps) {
+  const elementLocales = getElementLocales(element);
```

Comment:
Not sure how expensive this call is, but consider `useMemo` if worried

Reply:
I've taken a look and I'm not worried.

Reply:
@mpeyper  so should I update it with `useMemo` or there is no need to?

Reply:
No need to